AppCenterData/AppCenterData/MSBaseOptions.h

```diff
@@ -6,7 +6,7 @@
 @interface MSBaseOptions : NSObject
 
 /**
- * Device document time-to-live in seconds. Default is one day.
+ * Device document time-to-live in seconds. Default is infinite.
```

Comment:
Let's not hard code the values in comment as they are easy to get out of date.

```suggestion
 * Device document time-to-live in seconds.
```

Reply:
oh oh~ sure!